10-mac-config-git-globals

```diff
@@ -12,6 +7,10 @@ set -e
 usage() {
   cat << EOF
 Usage: $(basename $0) [-fh] [-e user.name] [-u user.email]
+This idempotent macOS Bash script configures git globals
+user name and password, ~/.gitignore and custom settings
+
+OPTIONS:
```

Comment:
Just adds text in the usage HEREDOC.  Acceptable risk to just vet this via `./10-mac-config-git-globals -h` and
existing git globals with `./10-mac-config-git-globals`